lib/matplotlib/colorbar.py

```diff
@@ -577,30 +533,19 @@ def _get_ticker_locator_formatter(self):
         """
         locator = self.locator
         formatter = self.formatter
-        if locator is None:
-            if self.boundaries is None:
-                if isinstance(self.norm, colors.NoNorm):
```

Comment:
I seem to have managed to drop `NoNorm` which I guess means this section of code had no tests.  Anyone know what this is supposed to do?